src/lib/arch/aarch64/MicroDecoder.cc

```diff
@@ -10,8 +10,9 @@ std::unordered_map<uint32_t, std::vector<Instruction>>
     MicroDecoder::microDecodeCache;
 std::forward_list<InstructionMetadata> MicroDecoder::microMetadataCache;
 
-MicroDecoder::MicroDecoder(YAML::Node config)
-    : instructionSplit_(config["Core"]["Micro-Operations"].as<bool>()) {}
+MicroDecoder::MicroDecoder() {
+  config::SimInfo::getConfig()["Core"]["Micro-Operations"] >> instructionSplit_;
+}
```

Comment:
use of `getValue` would allow `instructionSplit_` to be defined in the initialiser list again

Reply:
Unlike other instances commented on, changing this one now as it allows for instructionSplit_ to be `const`